src/main/resources/standardTestCases/non_root_default_user/non_root_default_user.wdl

```diff
@@ -0,0 +1,21 @@
+task notroot {
+  command {
+    echo $HOME
+  }
+
+  runtime {
+    docker: "mcovarr/notroot:v1"
```

Comment:
I haven't fully thought it out, but I've always had the warm and fuzzies for docker repos that [link to a github repo](http://stackoverflow.com/questions/27679740/how-to-add-dockerfile-to-docker-registry-page), no matter how small, as it allows the Dockerfile to be visible in hub.docker.com.

If we don't want a whole github repo, throwing out that the Dockerfile could live here next to the wdl in this folder, maybe:

```
src/main/resources/standardTestCases/non_root_default_user/docker/notroot/DockerFile
```

Would be happy if the more veteran centaur devs have a better location / subfolder / etc.